subpar.bzl

```diff
@@ -137,6 +139,7 @@ parfile_attrs = {
         cfg = "host",
     ),
     "compiler_args": attr.string_list(default = []),
+    "interpreter": attr.string(),
```

Comment:
please add  at lines 207 and 237
```
    interpreter = kwargs.pop("interpreter", None)
```
and  at lines 219 and 248 
```
        interpreter = interpreter,
```

otherwise `interpreter`  argument is propagated into py_binary/py_test rules

Reply:
@oxidase done